deploy/arm/testing-infrastructure/azuredeploy.json

```diff
@@ -4,12 +4,16 @@
     "parameters": {
         "EventGrid.Topic.Name": {
             "type": "string",
+            "minLength": 3,
```

Comment:
Ah cool. Where do these threshold values come from? 3 & 50?

Reply:
I tried it in the portal until they broke and gave me a value.

Probably documented somewhere but couldn't find it..